Pages/Login/login.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Veekay Cart | Login</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet">
+    <link rel="stylesheet" href="https://veekay-cmplib.netlify.app/CSS/components.css">
+    <link rel="stylesheet" href="https://veekay-cmplib.netlify.app/CSS/header.css">
+    <link rel="stylesheet" href="https://veekay-cmplib.netlify.app/CSS/list.css">
+    <link rel="stylesheet" href="/CSS/header.css">
+    <!-- <link rel="stylesheet" href="/Pages/Product/product.css"> -->
```

Comment:
Import all these CSS into single file and import that file.

Reply:
Sure Will do that.....after completing whole...will definately rrefactor